Kitodo/src/main/webapp/pages/massImport.xhtml

```diff
@@ -39,7 +39,7 @@
                   styleClass="secondary"/>
         <p:commandButton id="importButton"
                          value="#{msgs.massImport}"
-                         disabled="#{empty MassImportForm.records or not MassImportForm.getRulesetConfigurationForOpacImportComplete()}"
+                         disabled="#{empty MassImportForm.records or not MassImportForm.getRulesetConfigurationForOpacImportComplete() or not MassImportForm.firstColumnContainsRecordIdentifierOrDocType()}"
```

Comment:
I stumbled over this already, as i - after first put 'ID' in the column - put 'recordIdentifier' in the first column. After reading the description again i got that you have the name of the actual ruleset field which has the functional key applied. Probably just a matter of documentation, but it would be great if there was feedback, why exactly the button is disabled, some message like:

"The metadatum in the first column is not marked as a functional metadata of type `docType` or `recordIdentifier`.

Reply:
nevermind, sorry, it works.